lib/TableUtils.lua

```diff
@@ -332,4 +332,20 @@ function TableUtils.serialize(input, serializer)
 			"}"
 end
 
+function TableUtils.concat(...)
+	local result = {}
+	for i = 1, select("#", ...) do
+		local x = select(i, ...)
+		if type(x) == "table" then
+			for i, y in ipairs(x) do
```

Comment:
I'd generally avoid shadowing a variable in an outer loop. You're not even using it here, so could use `_`.

Reply:
done